### PR TITLE
Update drupal

### PR DESCRIPTION
**Pull request by tianon**
Changes:

- https://github.com/docker-library/drupal/commit/66ba6c9: Merge pull request https://github.com/docker-library/drupal/pull/278 from LaurentGoderre/alias-10.4
- https://github.com/docker-library/drupal/commit/92792f8: Set the v10 alias to 10.4
- https://github.com/docker-library/drupal/commit/cd49571: Remove more EOL bits (10.2, RCs that are now GA)
- https://github.com/docker-library/drupal/commit/897b539: Merge pull request https://github.com/docker-library/drupal/pull/277 from LaurentGoderre/alias-11.1
- https://github.com/docker-library/drupal/commit/89c6685: Set the 11.1 alias to 11 and latest
- https://github.com/docker-library/drupal/commit/fc6b030: Update 10.4-rc
- https://github.com/docker-library/drupal/commit/838c143: Update 10.4 to 10.4.0, composer 2
- https://github.com/docker-library/drupal/commit/1c78851: Update 10.2
- https://github.com/docker-library/drupal/commit/0865757: Update 11.1-rc
- https://github.com/docker-library/drupal/commit/dfb409a: Update 11.1 to 11.1.0, composer 2

### CODE REVIEW

**Review by LaurentGoderre — APPROVED** [review]

File: library/drupal [hunk 1]
```diff
@@ -4,35 +4,35 @@ Maintainers: Tianon Gravi <admwiggin@gmail.com> (@tianon),
...
 
-Tags: 11.1.0-rc1-php8.3-fpm-alpine3.20, 11.1-rc-php8.3-fpm-alpine3.20, 11.1.0-rc1-fpm-alpine3.20, 11.1-rc-fpm-alpine3.20
+Tags: 11.1.0-php8.3-fpm-alpine3.20, 11.1-php8.3-fpm-alpine3.20, 11.1.0-fpm-alpine3.20, 11.1-fpm-alpine3.20
 Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le, riscv64, s390x
-GitCommit: 561a5f6557e485370c336becc01e9bdb7ef582ac
-Directory: 11.1-rc/php8.3/fpm-alpine3.20
+GitCommit: dfb409af0becc76411b367ff2107209d7a1f3611
+Directory: 11.1/php8.3/fpm-alpine3.20
 
 Tags: 11.0.9-php8.3-apache-bookworm, 11.0-php8.3-apache-bookworm, 11-php8.3-apache-bookworm, php8.3-apache-bookworm, 11.0.9-php8.3-apache, 11.0-php8.3-apache, 11-php8.3-apache, php8.3-apache, 11.0.9-php8.3, 11.0-php8.3, 11-php8.3, php8.3, 11.0.9-apache-bookworm, 11.0-apache-bookworm, 11-apache-bookworm, apache-bookworm, 11.0.9-apache, 11.0-apache, 11-apache, apache, 11.0.9, 11.0, 11, latest
```
Comment (yosifkit) [inline library/drupal:37]: `latest` and `11` should move to `11.1.x` now, right?

Comment (LaurentGoderre) [reply library/drupal:37]: https://github.com/docker-library/drupal/pull/277

**Review by LaurentGoderre — DISMISSED** [review]

File: library/drupal (hunk 1 above)
Comment (LaurentGoderre) [reply library/drupal]: AH man, now I have to do the same for 10.4

Comment (LaurentGoderre) [reply library/drupal]: https://github.com/docker-library/drupal/pull/278

**Pull request merged**